src/stylesheets/core/_variables.scss

```diff
@@ -777,18 +789,20 @@ calculated variables
 ----------------------------------------
 */
 
-$theme-site-margins:                  4 !default;
-$theme-site-margins-mobile:           2 !default;
+$theme-site-margins: units($theme-site-margins);
+$theme-site-margins-mobile: units($theme-site-margins-mobile);
+$theme-nav-width: units($theme-nav-width);
+$theme-site-max-width: units($theme-site-max-width);
 
 $text-width-sm: 60ch;
 
 $text-max-width:                66ch !default; // 66 characters per line
 $text-narrow-width:             40ch !default;
 $text-wide-width:               72ch !default;
 $lead-max-width:                77rem !default;
-$site-max-width:                1040px !default;
-$site-margins:                  3rem !default;
-$site-margins-mobile:           1.5rem !default;
+$site-max-width:                $theme-site-max-width;
+$site-margins:                  $theme-site-margins;
+$site-margins-mobile:           $theme-site-margins-mobile;
 $article-max-width:             600px !default;
```

Comment:
I guess these should be OK to replace with new vars... I thought I'd reverted this, tho.
This means we can replace all instances of `$site-margins` and `$site-margins-mobile` with there `$theme-` counterparts.